### ISSUE TITLE
Not able to import the WSDL downloaded from Developer Portal into Visual Studio

### ISSUE THREAD

**Issue by prasadkkaranth** [Azure/api-management-developer-portal#2249]
We are not able to import the WSDL downloaded from Developer Portal into Visual Studio 2022. Getting the following error. Both Azure APIM and Visual Studio are owned by Microsoft, yet there is no seamless integration between these two, I am speechless. Please check.

**Sample Error:**
![image](https://github.com/Azure/api-management-developer-portal/assets/91755247/66cfb8f4-1832-492f-bd8b-8c6b668dafa1)

**Error snippet:**

`There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 13 Column: 3\r\n   Validation Error: The attributeGroup '[http://www.w3.org/XML/1998/namespace:specialAttrs'](http://www.w3.org/XML/1998/namespace:specialAttrs%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 12 Column: 3\r\n   Validation Error: The global attribute '[http://www.w3.org/XML/1998/namespace:id'](http://www.w3.org/XML/1998/namespace:id%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 11 Column: 3\r\n   Validation Error: The global attribute '[http://www.w3.org/XML/1998/namespace:base'](http://www.w3.org/XML/1998/namespace:base%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 3 Column: 3\r\n   Validation Error: The global attribute '[http://www.w3.org/XML/1998/namespace:space'](http://www.w3.org/XML/1998/namespace:space%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 2 Column: 3\r\n   Validation Error: The global attribute '[http://www.w3.org/XML/1998/namespace:lang'](http://www.w3.org/XML/1998/namespace:lang%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 3 Column: 6\r\n   Validation Error: The global element '[http://www.w3.org/2001/XMLSchema:schema'](http://www.w3.org/2001/XMLSchema:schema%27) has already been declared.
There was a validation error on a schema generated during export:\r\n    Source: \r\n    Line: 295 Column: 4\r\n   Validation Error: The identity constraint '[http://www.w3.org/2001/XMLSchema:identityConstraint'](http://www.w3.org/2001/XMLSchema:identityConstraint%27) has already been declared.`

**How to Reproduce**
Azure APIM Developer Portal -> Login -> APIs -> Search for a SOAP based API -> Click on 'API Definition' drop down and choose 'WSDL'

![image](https://github.com/Azure/api-management-developer-portal/assets/91755247/d97ba91c-7ada-407b-ad30-484935924f02)

Now open Visual Studio 2022, choose a project and Right click and choose 'Add Service Reference' , you should get an error similar to the above mentioned one.

**Comment by malincrist** [comment]
Hello @prasadkkaranth , I was not able to reproduce the issue you described.
It appears that there is a problem with the schema itself. Could you please share the schema to [apimportalfeedback@microsoft.com](mailto:apimportalfeedback@microsoft.com) so we can investigate this further?

**Comment by prasadkkaranth** [comment]
I have sent the wsdl for your review. I downloaded it from APIM Developer Portal, like the way I mentioned above. Please check. Thank you!

**Comment by prasadkkaranth** [comment]
Any updates on this ?

**Comment by malincrist** [comment]
@prasadkkaranth , apologies for the late reply.
I tried two different scenarios:
1. I directly imported the schema you provided in VS2022 and got the error.
2. I imported the schema  to APIM, then downloaded it from Developer Portal, then tried to upload it in VS2022 and got the same error.

It appears that the issue you are facing is not related to Developer Portal, nor APIM. It seems that there is a issue with the schema itself so I suggest you check that it is correct or reach out to the tool that generated it and ask for assistance.
